geco/mips/miplib/loader.py

```diff
@@ -0,0 +1,71 @@
+import tempfile
+import requests
```

Comment:
requests is not part of the standard library, is it?
Is 'urllib' able to do the same job, or shall we add this to our environment?

Reply:
I am very glad you said this, the urllib code is even more readable.